deploy.js

```diff
@@ -0,0 +1,62 @@
+var argv = require('yargs').argv;
+var child = require('child_process');
+var replace = require('replace-in-file');
+var FtpDeploy = require("ftp-deploy");
+var ftpDeploy = new FtpDeploy();
+
+function execSync(command) {
+    // console.info('Exec command: ' + command);
+    var res = child.execSync(command);
+    res = String(res).trim();
+    return res;
+}
+
+// Replace <!-- commit-info --> in samples/dash-if-reference-player/index.html with git branch and commit
+var branch = execSync('git rev-parse --abbrev-ref HEAD');
+var hash = execSync('git rev-parse HEAD');
+
+var str = `(${branch}, commit: <a href="https://github.com/Dash-Industry-Forum/dash.js/commit/${hash}">${hash.substring(0, 8)}</a>)`;
+
+try {
+    replace.sync({
+        files: 'samples/dash-if-reference-player/index.html',
+        from: '<!-- commit-info -->',
+        to: str
+    });
+} catch (e) {
+    console.error('Failed to replace git info: ', e);
+}
+
+// Push files ont ftp server
+var config = {
+    user: argv.user,
+    password: argv.password,
+    host: argv.host,
+    port: argv.port,
+    localRoot: __dirname + "../",
+    remoteRoot: '/dummy',
```

Comment:
Remote path for nightly deploys is '/'

Reply:
Yes I know, as already said I modified some data temporarily in order to check if ftp push is still successfull using this script, but without overwriting existing files (development branch files) on ftp server